ddl/ddl.go

```diff
@@ -976,9 +977,13 @@ func getDDLJobsInQueue(t *meta.Meta, jobListKey meta.JobListKeyType) ([]*model.J
 	return jobs, nil
 }
 
-// GetDDLJobs get all DDL jobs and sorts jobs by job.ID.
-func GetDDLJobs(txn kv.Transaction) ([]*model.Job, error) {
-	t := meta.NewMeta(txn)
+// GetAllDDLJobs get all DDL jobs and sorts jobs by job.ID.
+func GetAllDDLJobs(t *meta.Meta) ([]*model.Job, error) {
```

Comment:
Why are we doing this? I don't feel the need

Reply:
well, it is like an entrance of "get all ddl jobs", it now gets from `meta`, in the future, we may add another way to get them and use a switch to control it.

Reply:
I think this can be changed in the future, there's no need to add the function of `getDDLJobs`.

Reply:
Done, PTAL